raycicmd/converter.go

```diff
@@ -8,6 +8,7 @@ import (
 )
 
 const windowsJobEnv = "WINDOWS"
+const macosJobEnv = "MACOS"
```

Comment:
could you add both macos intel and apple silicon in this PR?

Reply:
all machine type can share this same macosJobEnv right?

Reply:
ah, that might be right.

then could you add the other `instance_type` in the `macos.rayci.yaml`?